rector-prefixed 0.7 throw '...PhpStormStubsMap' errors

<!-- First, thank you for reporting a bug. That takes time and we appreciate that! -->

| Subject        | Details                                                               |
| :------------- | :-------------------------------------------------------------------- |
| Rector version | v0.7                   |
| PHP version    | PHP 7.3.11      |
| Full Command   | vendor/bin/rector process web/modules/contrib/config_filter --set dead-code --dry-run           | 
| Demo link      |  | 
| rector.yaml    | (see below) | 
```
parameters:
  autoload_paths:
    - 'web/core'
    - 'web/core/modules'
    - 'web/modules'
    - 'web/profiles'

  file_extensions:
    - module
    - theme
    - install
    - profile
    - inc
    - engine
```
## Current Behaviour 

Installing Drupal8 and adding rector-prefixed 0.6.13 works as expected
Steps to replicate:
- `composer create-project drupal/recommended-project drupal8`
- `cd drupal8`
- `composer require rector/rector-prefixed:0.6.13 --dev`
- (create the rector.yaml above)
- `vendor/bin/rector process web/core/modules/aggregator/src/Plugin/views/row/Rss.php --set deadcode --dry-run`

The output is correct and it display the diff of berfore/after the rector
```
1 file with changes
===================

1) web/core/modules/aggregator/src/Plugin/views/row/Rss.php

    ---------- begin diff ----------
--- Original
+++ New
@@ -66,14 +66,12 @@
         'attributes' => ['isPermaLink' => 'false'],
       ],
     ];
-
-    $build = [
+    return [
       '#theme' => $this->themeFunctions(),
       '#view' => $this->view,
       '#options' => $this->options,
       '#row' => $item,
     ];
-    return $build;
   }

 }
    ----------- end diff -----------

Applied rules:

 * Rector\CodeQuality\Rector\Return_\SimplifyUselessVariableRector
```

But upgrading to rector-prefix v0.7 is causing problems:
- `composer require rector/rector-prefixed:0.7 --dev`
- `vendor/bin/rector process web/core/modules/aggregator/src/Plugin/views/row/Rss.php --set deadcode --dry-run`

```                                                                                                             
 [ERROR] Could not process "web/core/modules/aggregator/src/Plugin/views/row/Rss.php" file, due to: "Class '_HumbugBox46c80243413c\JetBrains\PHPStormStub\PhpStormStubsMap' not found".                            
                                                                                                                
 [ERROR] Could not process "web/core/modules/aggregator/src/Plugin/views/row/Rss.php" file, due to: "Call to a member function hasByType() on null".                                                               
```

**report from command line with --debug**
`vendor/bin/rector process web/core/modules/aggregator/src/Plugin/views/row/Rss.php --set deadcode --dry-run --debug`
```
File "/Users/shaal/code/test/my_site_name_dir/vendor/autoload.php" is about to be loaded in "_HumbugBox46c80243413c\AutoloadIncluder::includeCwdVendorAutoloadIfExists()" on line 53"
File "/Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/../../autoload.php" is about to be loaded in "_HumbugBox46c80243413c\AutoloadIncluder::autoloadProjectAutoloaderFile()" on line 86"
File "phar://rector.phar/vendor/autoload.php" is about to be loaded in "_HumbugBox46c80243413c\AutoloadIncluder::includeDependencyOrRepositoryVendorAutoloadIfExists()" on line 65"
File "/Users/shaal/code/test/my_site_name_dir/vendor/autoload.php" is about to be loaded in "_HumbugBox46c80243413c\AutoloadIncluder::includeCwdVendorAutoloadIfExists()" on line 53"
File "/Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/../../autoload.php" is about to be loaded in "_HumbugBox46c80243413c\AutoloadIncluder::autoloadProjectAutoloaderFile()" on line 86"
File "phar://rector.phar/vendor/autoload.php" is about to be loaded in "_HumbugBox46c80243413c\AutoloadIncluder::includeDependencyOrRepositoryVendorAutoloadIfExists()" on line 65"
Rector v0.7.0
Config file: rector.yml

[parsing] web/core/modules/aggregator/src/Plugin/views/row/Rss.php
PHP Fatal error:  Uncaught Error: Class '_HumbugBox46c80243413c\JetBrains\PHPStormStub\PhpStormStubsMap' not found in phar:///Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/rector/vendor/ondrejmirtes/better-reflection/src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php:67
Stack trace:
#0 phar:///Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/rector/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Type/PhpInternalSourceLocator.php(38): _HumbugBox46c80243413c\Roave\BetterReflection\SourceLocator\SourceStubber\PhpStormStubsSourceStubber->generateClassStub('key')
#1 phar:///Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/rector/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Type/PhpInternalSourceLocator.php(31): _HumbugBox46c80243413c\Roave\BetterReflection\SourceLocator\Type\PhpInternalSourceLocator->getClassSource(Object(_HumbugBox46c80243413c\Roave\BetterReflection\Identifier\Identifier))
#2 phar:///Users/shaal/code/test/my_site_name_dir in phar:///Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/rector/vendor/ondrejmirtes/better-reflection/src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php on line 67
Fatal error: Uncaught Error: Class '_HumbugBox46c80243413c\JetBrains\PHPStormStub\PhpStormStubsMap' not found in phar:///Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/rector/vendor/ondrejmirtes/better-reflection/src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php:67
Stack trace:
#0 phar:///Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/rector/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Type/PhpInternalSourceLocator.php(38): _HumbugBox46c80243413c\Roave\BetterReflection\SourceLocator\SourceStubber\PhpStormStubsSourceStubber->generateClassStub('key')
#1 phar:///Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/rector/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Type/PhpInternalSourceLocator.php(31): _HumbugBox46c80243413c\Roave\BetterReflection\SourceLocator\Type\PhpInternalSourceLocator->getClassSource(Object(_HumbugBox46c80243413c\Roave\BetterReflection\Identifier\Identifier))
#2 phar:///Users/shaal/code/test/my_site_name_dir in phar:///Users/shaal/code/test/my_site_name_dir/vendor/rector/rector-prefixed/rector/vendor/ondrejmirtes/better-reflection/src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php on line 67
```
<!-- description/screenshot -->

## Minimal PHP Code Causing Issue

<!-- if possible, always run Rector on specfic file, that causes the error -->
<!-- e.g. vendor/bin/rector process /src/SpecificFile.php --set dead-code --debug -->  

```php
<?php 

// put file content here
```

## Expected Behaviour

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,12 +14,17 @@ jobs:
         # "Deploy to https://github.com/rectorphp/rector-prefixed"
         -
             name: Compile and Release rector.phar
-            if: (branch = master OR tag IS present) && type = push
+#            if: (branch = master OR tag IS present) && type = push
             php: 7.2
             script:
                 - cd compiler
+
+                # see https://gist.github.com/jeffersonmartin/d0d4a8dfec90d224d14f250b36c74d2f
+                # see https://www.previousnext.com.au/blog/managing-composer-github-access-personal-access-tokens
+                - composer config -g github-oauth.github.com $GITHUB_TOKEN
+
                 - composer install
-                - bin/compile
+                - bin/compile --ansi
                 - ../tmp/rector.phar
 
                 - cd ..

--- a/compiler/bin/compile
+++ b/compiler/bin/compile
@@ -3,19 +3,27 @@
 
 declare(strict_types = 1);
 
-use Rector\Compiler\Composer\ComposerJsonManipulator;
-use Rector\Compiler\Console\CompileCommand;
-use Symfony\Component\Console\Application;
-use Symfony\Component\Filesystem\Filesystem;
+use Psr\Container\ContainerInterface;
+use Rector\Compiler\Console\RectorCompilerConsoleApplication;
+use Rector\Compiler\HttpKernel\RectorCompilerKernel;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-// dependency injection for poor people
-$filesystem = new Filesystem();
-$composerJsonManipulator = new ComposerJsonManipulator($filesystem);
-$compileCommand = new CompileCommand(__DIR__ . '/../build', __DIR__ . '/../..', $composerJsonManipulator);
+final class ContainerFactory
+{
+    public function create(): ContainerInterface
+    {
+        $environment = 'prod' . random_int(1, 10000000);
+        $rectorCompilerKernel = new RectorCompilerKernel($environment, true);
+        $rectorCompilerKernel->boot();
 
-$application = new Application();
-$application->add($compileCommand);
-$application->setDefaultCommand($compileCommand->getName(), true);
-$application->run();
+        return $rectorCompilerKernel->getContainer();
+    }
+}
+
+$containerFactory = new ContainerFactory();
+$container = $containerFactory->create();
+
+/** @var RectorCompilerConsoleApplication $application */
+$application = $container->get(RectorCompilerConsoleApplication::class);
+exit($application->run());

--- a/compiler/composer.json
+++ b/compiler/composer.json
@@ -5,19 +5,18 @@
     "require": {
         "php": "^7.2",
         "nette/neon": "^3.0",
+        "nette/utils": "^3.0",
+        "sebastian/diff": "^3.0|^4.0",
         "symfony/console": "^4.4|^5.0",
         "symfony/process": "^4.4|^5.0",
         "symfony/filesystem": "^4.4|^5.0",
         "symfony/finder": "^4.4|^5.0",
-        "symplify/package-builder": "^7.2",
-        "nette/utils": "^3.0"
+        "symplify/auto-bind-parameter": "^7.2",
+        "symplify/package-builder": "^7.2"
     },
     "autoload": {
         "psr-4": {
             "Rector\\Compiler\\": "src"
         }
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^8.4"
     }
 }

--- a/compiler/config/config.yaml
+++ b/compiler/config/config.yaml
@@ -1,0 +1,22 @@
+parameters:
+    dataDir: '%kernel.project_dir%/build'
+    buildDir: '%kernel.project_dir%/..'
+
+services:
+    _defaults:
+        public: true
+        autowire: true
+
+    Rector\Compiler\:
+        resource: '../src'
+        exclude:
+            - '../src/Exception/*'
+            - '../src/HttpKernel/*'
+
+    Symplify\PackageBuilder\Console\Style\SymfonyStyleFactory: null
+    Symfony\Component\Console\Style\SymfonyStyle:
+        factory: ['@Symplify\PackageBuilder\Console\Style\SymfonyStyleFactory', 'create']
+
+    SebastianBergmann\Diff\Differ: null
+
+    Symfony\Component\Filesystem\Filesystem: null

--- a/compiler/src/Composer/ComposerJsonManipulator.php
+++ b/compiler/src/Composer/ComposerJsonManipulator.php
@@ -6,6 +6,7 @@ namespace Rector\Compiler\Composer;
 
 use Nette\Utils\FileSystem as NetteFileSystem;
 use Nette\Utils\Json;
+use Rector\Compiler\Differ\ConsoleDiffer;
 use Symfony\Component\Filesystem\Filesystem;
 
 final class ComposerJsonManipulator
@@ -20,9 +21,15 @@ final class ComposerJsonManipulator
      */
     private $originalComposerJsonFileContent;
 
-    public function __construct(Filesystem $filesystem)
+    /**
+     * @var ConsoleDiffer
+     */
+    private $consoleDiffer;
+
+    public function __construct(Filesystem $filesystem, ConsoleDiffer $consoleDiffer)
     {
         $this->filesystem = $filesystem;
+        $this->consoleDiffer = $consoleDiffer;
     }
 
     public function fixComposerJson(string $composerJsonFile): void
@@ -41,6 +48,9 @@ final class ComposerJsonManipulator
         $json = $this->addAllowDevPackages($json);
 
         $encodedJson = Json::encode($json, Json::PRETTY);
+
+        // show diff
+        $this->consoleDiffer->diff($this->originalComposerJsonFileContent, $encodedJson);
 
         $this->filesystem->dumpFile($composerJsonFile, $encodedJson);
     }

--- a/compiler/src/Console/Command/CompileCommand.php
+++ b/compiler/src/Console/Command/CompileCommand.php
@@ -2,9 +2,10 @@
 
 declare(strict_types=1);
 
-namespace Rector\Compiler\Console;
+namespace Rector\Compiler\Console\Command;
 
 use Rector\Compiler\Composer\ComposerJsonManipulator;
+use Rector\Compiler\Debug\FileLister;
 use Rector\Compiler\Renaming\JetbrainsStubsRenamer;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
@@ -12,7 +13,6 @@ use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 use Symfony\Component\Process\Process;
 use Symplify\PackageBuilder\Console\ShellCode;
-use Symplify\PackageBuilder\Console\Style\SymfonyStyleFactory;
 
 /**
  * Inspired by @see https://github.com/phpstan/phpstan-src/blob/f939d23155627b5c2ec6eef36d976dddea22c0c5/compiler/src/Console/CompileCommand.php
@@ -44,36 +44,45 @@ final class CompileCommand extends Command
      */
     private $jetbrainsStubsRenamer;
 
-    public function __construct(string $dataDir, string $buildDir, ComposerJsonManipulator $composerJsonManipulator)
-    {
-        parent::__construct();
+    /**
+     * @var FileLister
+     */
+    private $fileLister;
 
-        $this->composerJsonManipulator = $composerJsonManipulator;
-
+    public function __construct(
+        string $dataDir,
+        string $buildDir,
+        ComposerJsonManipulator $composerJsonManipulator,
+        SymfonyStyle $symfonyStyle,
+        JetbrainsStubsRenamer $jetbrainsStubsRenamer,
+        FileLister $fileLister
+    ) {
         $this->dataDir = $dataDir;
         $this->buildDir = $buildDir;
 
-        $symfonyStyleFactory = new SymfonyStyleFactory();
-        $this->symfonyStyle = $symfonyStyleFactory->create();
+        $this->composerJsonManipulator = $composerJsonManipulator;
+        $this->jetbrainsStubsRenamer = $jetbrainsStubsRenamer;
+        $this->fileLister = $fileLister;
+        $this->symfonyStyle = $symfonyStyle;
 
-        $this->jetbrainsStubsRenamer = new JetbrainsStubsRenamer($this->symfonyStyle);
+        parent::__construct();
     }
 
     protected function configure(): void
     {
-        $this->setName('rector:compile');
+        $this->setName(self::class);
         $this->setDescription('Compile prefixed rector.phar');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        // 1.
         $composerJsonFile = $this->buildDir . '/composer.json';
-
-        $this->symfonyStyle->note('Loading and updating ' . $composerJsonFile);
+        $this->symfonyStyle->section('Loading and updating ' . $composerJsonFile);
         $this->composerJsonManipulator->fixComposerJson($composerJsonFile);
 
-        $this->symfonyStyle->note('Updating root rector/rector dependencies, without require-dev');
-
+        // 2.
+        $this->symfonyStyle->section('Updating root rector/rector dependencies, without require-dev');
         // @see https://github.com/dotherightthing/wpdtrt-plugin-boilerplate/issues/52
         $process = new Process([
             'composer',
@@ -83,28 +92,29 @@ final class CompileCommand extends Command
             '--no-interaction',
             '--classmap-authoritative',
         ], $this->buildDir, null, null, null);
+        $process->mustRun(static function (string $type, string $buffer) use ($output): void {
+            $output->write($buffer);
+        });
 
         // debug
-        $process = new Process(['ls', '-l', 'vendor'], $this->buildDir);
-        $process->mustRun(static function (string $type, string $buffer) use ($output): void {
-            $output->write($buffer);
-        });
+        $this->fileLister->listFilesInDirectory($this->buildDir . '/vendor/jetbrains');
 
-        $this->symfonyStyle->note('Renaming PHPStorm stubs from "*.php" to ".stub"');
+        // 3.
+        $this->symfonyStyle->section('Renaming PHPStorm stubs from "*.php" to ".stub"');
         $this->jetbrainsStubsRenamer->renamePhpStormStubs($this->buildDir);
 
-        $process->mustRun(static function (string $type, string $buffer) use ($output): void {
-            $output->write($buffer);
-        });
-
+        // 4.
         // the '--no-parallel' is needed, so "scoper.php.inc" can "require __DIR__ ./vendor/autoload.php"
         // and "Nette\Neon\Neon" class can be used there
+        $this->symfonyStyle->section('Packing and prefixing rector.phar with Box and PHP Scoper');
         $process = new Process(['php', 'box.phar', 'compile', '--no-parallel'], $this->dataDir, null, null, null);
-
         $process->mustRun(static function (string $type, string $buffer) use ($output): void {
             $output->write($buffer);
         });
 
+        // 5.
+        $this->symfonyStyle->section('Restoring root composer.json with "require-dev"');
+        $this->symfonyStyle->note('You still need to run "composer update" to install those dependencies');
         $this->composerJsonManipulator->restoreComposerJson($composerJsonFile);
 
         return ShellCode::SUCCESS;

--- a/compiler/src/Console/RectorCompilerConsoleApplication.php
+++ b/compiler/src/Console/RectorCompilerConsoleApplication.php
@@ -1,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Compiler\Console;
+
+use Rector\Compiler\Console\Command\CompileCommand;
+use Symfony\Component\Console\Application;
+
+final class RectorCompilerConsoleApplication extends Application
+{
+    public function __construct(CompileCommand $compileCommand)
+    {
+        parent::__construct('Rector Compiler', 'v1.0');
+
+        $this->add($compileCommand);
+        $this->setDefaultCommand(get_class($compileCommand), true);
+    }
+}

--- a/compiler/src/Debug/FileLister.php
+++ b/compiler/src/Debug/FileLister.php
@@ -1,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Compiler\Debug;
+
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+final class FileLister
+{
+    /**
+     * @var SymfonyStyle
+     */
+    private $symfonyStyle;
+
+    public function __construct(SymfonyStyle $symfonyStyle)
+    {
+        $this->symfonyStyle = $symfonyStyle;
+    }
+
+    public function listFilesInDirectory(string $directory): void
+    {
+        $finder = (new Finder())
+            ->in($directory)
+            ->name('*.php')
+            ->files();
+
+        /** @var SplFileInfo[] $phpFileInfos */
+        $phpFileInfos = $finder->getIterator();
+
+        $this->symfonyStyle->newLine();
+        $this->symfonyStyle->section(sprintf('Files found in "%s" directory', $directory));
+
+        foreach ($phpFileInfos as $phpFileInfo) {
+            $this->symfonyStyle->writeln($phpFileInfo->getRelativePathname());
+        }
+
+        $this->symfonyStyle->newLine();
+    }
+}

--- a/compiler/src/Differ/ConsoleDiffer.php
+++ b/compiler/src/Differ/ConsoleDiffer.php
@@ -1,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Compiler\Differ;
+
+use SebastianBergmann\Diff\Differ;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+final class ConsoleDiffer
+{
+    /**
+     * @var Differ
+     */
+    private $differ;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $symfonyStyle;
+
+    /**
+     * @var ConsoleDifferFormatter
+     */
+    private $consoleDifferFormatter;
+
+    public function __construct(
+        SymfonyStyle $symfonyStyle,
+        Differ $differ,
+        ConsoleDifferFormatter $consoleDifferFormatter
+    ) {
+        $this->symfonyStyle = $symfonyStyle;
+        $this->differ = $differ;
+        $this->consoleDifferFormatter = $consoleDifferFormatter;
+    }
+
+    public function diff(string $old, string $new): void
+    {
+        $diff = $this->differ->diff($old, $new);
+        $consoleFormatted = $this->consoleDifferFormatter->format($diff);
+        $this->symfonyStyle->writeln($consoleFormatted);
+    }
+}

--- a/compiler/src/Differ/ConsoleDifferFormatter.php
+++ b/compiler/src/Differ/ConsoleDifferFormatter.php
@@ -1,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Compiler\Differ;
+
+use Nette\Utils\Strings;
+use Symfony\Component\Console\Formatter\OutputFormatter;
+
+/**
+ * Mimics @see \Rector\ConsoleDiffer\Console\Formatter\DiffConsoleFormatter
+ */
+final class ConsoleDifferFormatter
+{
+    /**
+     * @var string
+     */
+    private $template;
+
+    public function __construct()
+    {
+        $this->template = sprintf(
+            '<comment>    ---------- begin diff ----------</comment>%s%%s%s<comment>    ----------- end diff -----------</comment>',
+            PHP_EOL,
+            PHP_EOL
+        );
+    }
+
+    public function format(string $diff): string
+    {
+        return $this->formatWithTemplate($diff, $this->template);
+    }
+
+    private function formatWithTemplate(string $diff, string $template): string
+    {
+        return sprintf($template, implode(PHP_EOL, array_map(function ($string): string {
+            // make "+" lines green
+            $string = Strings::replace($string, '#^(\+.*)#', '<fg=green>$1</fg=green>');
+            // make "-" lines red
+            $string = Strings::replace($string, '#^(\-.*)#', '<fg=red>$1</fg=red>');
+            // make "@ note" lines cyan
+            $string = Strings::replace($string, '#^(@.*)#', '<fg=cyan>$1</fg=cyan>');
+
+            if ($string === ' ') {
+                $string = rtrim($string);
+            }
+
+            return $string;
+        }, Strings::split(OutputFormatter::escape(rtrim($diff)), "#\n\r|\n#"))));
+    }
+}

--- a/compiler/src/HttpKernel/RectorCompilerKernel.php
+++ b/compiler/src/HttpKernel/RectorCompilerKernel.php
@@ -1,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Compiler\HttpKernel;
+
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Symfony\Component\HttpKernel\Kernel;
+use Symplify\AutoBindParameter\DependencyInjection\CompilerPass\AutoBindParameterCompilerPass;
+
+final class RectorCompilerKernel extends Kernel
+{
+    public function getCacheDir(): string
+    {
+        // manually configured, so it can be replaced in phar
+        return sys_get_temp_dir() . '/_rector_compiler';
+    }
+
+    public function getLogDir(): string
+    {
+        // manually configured, so it can be replaced in phar
+        return sys_get_temp_dir() . '/_rector_compiler_log';
+    }
+
+    public function registerContainerConfiguration(LoaderInterface $loader): void
+    {
+        $loader->load(__DIR__ . '/../../config/config.yaml');
+    }
+
+    /**
+     * @return BundleInterface[]
+     */
+    public function registerBundles(): array
+    {
+        return [];
+    }
+
+    protected function build(ContainerBuilder $containerBuilder): void
+    {
+        $containerBuilder->addCompilerPass(new AutoBindParameterCompilerPass());
+    }
+}

--- a/compiler/src/Renaming/JetbrainsStubsRenamer.php
+++ b/compiler/src/Renaming/JetbrainsStubsRenamer.php
@@ -38,10 +38,10 @@ final class JetbrainsStubsRenamer
         foreach ($stubFileInfos as $stubFileInfo) {
             $path = $stubFileInfo->getPathname();
 
+            $this->symfonyStyle->note(sprintf('Renaming "%s"', $stubFileInfo->getRelativePathname()));
+
             $filenameWithStubSuffix = dirname($path) . '/' . $stubFileInfo->getBasename('.php') . '.stub';
             FileSystem::rename($path, $filenameWithStubSuffix);
-
-            $this->symfonyStyle->note(sprintf('Renaming "%s"', $stubFileInfo->getRealPath()));
         }
     }
 

